src/ssl.c

```diff
@@ -10592,7 +10592,7 @@ int wolfSSL_DTLS_SetCookieSecret(WOLFSSL* ssl,
                 if (ssl->options.sendVerify) {
                     if ( (ssl->error = SendCertificate(ssl)) != 0) {
                     #ifdef WOLFSSL_CHECK_ALERT_ON_ERR
-                        ProcessReplyEx(ssl, 1); /* See if an alert was sent. */
+                        wolfSSL_maybeCheckAlertOnErr(ssl, ssl->error);
                     #endif
```

Comment:
I think it would be cleaner to move `WOLFSSL_CHECK_ALERT_ON_ERR` inside of `wolfSSL_maybeCheckAlertOnErr`. When `WOLFSSL_CHECK_ALERT_ON_ERR` is not defined then `wolfSSL_maybeCheckAlertOnErr` would be a no-op.